performance/swiftinterop/src/main/kotlin/org/jetbrains/multigraph/Multigraph.kt

```diff
@@ -0,0 +1,165 @@
+/*
```

Comment:
Kotlin code does a lot of heavy computations for simple operations. Not sure if a benchmark using these operations is suitable for measuring Swift -> Kotlin interop.

Reply:
I mostly tried to repeat and measure some scenarios that could be in real applications with swift-kotlin interop, when common part is written on kotlin and swift just used function to get some results to show in UI. I can add very small functions to try measure mostly interop layer. But I amn't sure if such microbenchmarks can be stable enough. Do you suggest just add or replace all?

Reply:
> I mostly tried to repeat and measure some scenarios that could be in real applications with swift-kotlin interop, when common part is written on kotlin

Makes sense.

> and swift just used function to get some results to show in UI.

This doesn't seem the only case to me.

> But I amn't sure if such microbenchmarks can be stable enough

Calling small Kotlin functions from Swift doesn't imply making a microbenchmark. You can have some non-trivial logic in Swift instead.

Reply:
I added 2 simple benchmarks, we already have similar small benchmarks in Ring, so seems that there shouldn't be problems.